semantic/src/site/globals/site.overrides

```diff
@@ -226,6 +238,38 @@ p strong {
   padding: 0;
 }
 
+/*--------------------------------
+      Application List
+---------------------------------*/
+
+#list-container {
+  padding: 25px 0;
+  max-width: 900px; // Maybe this could be wider if table is getting cluttered?
```

Comment:
Hmm, i think this should be ok for now, but was wondering why not 'width' 100%

Reply:
Deb's design is a 900px container within the larger container, so it has some margin at the sides:
![Screen Shot 2021-04-28 at 3 54 05 PM](https://user-images.githubusercontent.com/5456533/116344335-2c50e280-a83a-11eb-8c5d-f41cc78f9e14.png)